prisma/schema.prisma

```diff
@@ -40,28 +34,184 @@ model Account {
     @@unique([provider, providerAccountId])
 }
 
+model VerificationToken {
+    identifier String
+    token      String   @unique
+    expires    DateTime
+
+    @@unique([identifier, token])
+}
+
 model Session {
-    id           String   @id @default(cuid())
+    id           String   @id @default(uuid())
     sessionToken String   @unique
     userId       String
     expires      DateTime
     user         User     @relation(fields: [userId], references: [id], onDelete: Cascade)
 }
 
 model User {
-    id            String    @id @default(cuid())
```

Comment:
This led me down a rabbit hole reading up on uuid vs cuid vs nanoid.

Long story short, uuid is most robust but maybe overkill. Ultimately it is not an issue that will ever affect us and consistency wins.

Reply:
Yeah, I got a little lost in that rabbit hole, ultimately for our need I don't think it matters too much what we picked.